code/datums/mutations/holy_mutation/burdened.dm

```diff
@@ -0,0 +1,161 @@
+
+#define FIRST_TRUTH_REWARD 3
+#define SECOND_TRUTH_REWARD 6
+
+#define BAD_MUTATIONS_REQUIRED 3
+
+///Burdened grants some more mutations upon injuring yourself sufficiently
+/datum/mutation/human/burdened
+	name = "Burdened"
+	desc = "Less of a genome and of a forceful rewrite of genes. Nothing Nanotrasen supplies allows for a genetic restructure like this... \
+	The user feels compelled to injure themselves in various incapacitating and horrific ways. Oddly enough, this gene seems to be connected \
+	to several other ones, possibly ready to trigger more genetic changes in the future."
+	quality = POSITIVE //so it gets carried over on revives
+	locked = TRUE
+	text_gain_indication = "<span class='notice'>You feel burdened!</span>"
+	text_lose_indication = "<span class='warning'>You no longer feel the need to burden yourself!</span>"
+	var/burden_level = 0
+
+/datum/mutation/human/burdened/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	//two burdens - eyes and tongue
+	RegisterSignal(owner, COMSIG_CARBON_GAIN_ORGAN, .proc/organ_added_burden)
+	RegisterSignal(owner, COMSIG_CARBON_LOSE_ORGAN, .proc/organ_removed_burden)
+	//one burden - majority of limbs
+	RegisterSignal(owner, COMSIG_CARBON_ATTACH_LIMB, .proc/limbs_added_burden)
+	RegisterSignal(owner, COMSIG_CARBON_REMOVE_LIMB, .proc/limbs_removed_burden)
+	//one burden - an addiction
+	RegisterSignal(owner, COMSIG_CARBON_GAIN_ADDICTION, .proc/addict_added_burden)
+	RegisterSignal(owner, COMSIG_CARBON_LOSE_ADDICTION, .proc/addict_removed_burden)
+	//one burden - 3 major negative mutations
+	RegisterSignal(owner, COMSIG_CARBON_GAIN_MUTATION, .proc/mutation_added_burden)
+	RegisterSignal(owner, COMSIG_CARBON_LOSE_MUTATION, .proc/mutation_removed_burden)
+
+/datum/mutation/human/burdened/proc/update_burden(increase)
+	if(owner.dna)
+		var/datum/dna/woke_dna = owner.dna
+		if(burden_level >= FIRST_TRUTH_REWARD)
+			woke_dna.add_mutation(TELEPATHY)
+			woke_dna.add_mutation(MUT_MUTE)
```

Comment:
can genetics copy these mutations from you?

Reply:
No, they're other category. stuff like wizard mutate does the same category